composer.json

```diff
@@ -6,31 +6,41 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/emr": "^1.0-beta11",
```

Comment:
Use the dev branch where the upgrade has been done in order to test it.
The same for the multivalue_form_element component.